lib/model/classycat.py

```diff
@@ -1,13 +1,26 @@
-from typing import Union
+from typing import Union, Dict, Any, Optional, List
+from pydantic import BaseModel
 from lib.logger import logger
 from lib.model.model import Model
-from lib.schemas import Message, ClassyCatSchemaResponse, ClassyCatBatchClassificationResponse
+from lib.schemas import Message
 from lib.model.classycat_classify import Model as ClassifyModel
 from lib.model.classycat_schema_create import Model as ClassyCatSchemaCreateModel
 from lib.model.classycat_schema_lookup import Model as ClassyCatSchemaLookupModel
 from lib.base_exception import PrestoBaseException
 
 
+class ClassyCatResponse(BaseModel):
+    responseMessage: Optional[str] = None
+
+
+class ClassyCatBatchClassificationResponse(ClassyCatResponse):
+    classification_results: Optional[List[dict]] = []
```

Comment:
seems like should be non-optional? (must at least return an empty dict?)

Reply:
well, I think I mostly agree with you. however these types are only enforced upon creation of these objects, when the response object is just an empty object waiting to be populated by the `process()` method. if we lift the optional option, then we would have to populate them on creation or have a default. I agree that down the line we want do better typing but for now this seems like it doesn't fit our message processing model without some major refactoring work.

Reply:
seems like the current default of `[]` would work?

Reply:
yes, it will work for this specific case, I have updated and now they are in `classycat_response.py`. but I think overall this is a subtle problem with our specific use of pydantic in presto.